### ISSUE TITLE
Feature request: Ability to report posts/users/instances

### ISSUE THREAD

**Issue by HelmicNewciv** [RasmusLindroth/tut#186]
Ideally the user should be able to do everything a user on a full-fat Mastodon interface can do to report a post - flag relevant posts by the reported user (including an ability to view the media of posts to find the correct one), leave a comment, choose whether to notify the other instance's moderation team, and so on.

**Comment by RasmusLindroth** [comment]
I agree that this should be possible. I've some features I want to implement first, so this isn't my priority but I'll get to this issue eventually. Thank you for giving me suggestions of features you'd like to see in tut

**Comment by HelmicNewciv** [comment]
Oh, and a way to easily view the other instance's homepage/about or whatever, to make sure you're not sending a report to an instance that's going to give you shit about it.
